packages/basemap/pyproject.toml

```diff
@@ -2,14 +2,7 @@
 requires = [
     'setuptools',
     'wheel',
-    'numpy == 1.26.1; python_version >= "3.12"',
-    'numpy == 1.23.3; python_version == "3.11"',
-    'numpy == 1.21.4; python_version == "3.10"',
-    'numpy == 1.21.4; sys_platform == "darwin" and (python_version >= "3.7" and python_version <= "3.9")',
-    'numpy == 1.16.6; sys_platform != "darwin" and (python_version >= "3.7" and python_version <= "3.9")',
-    'numpy == 1.16.6; python_version == "2.7" or (python_version >= "3.4" and python_version <= "3.6")',
-    'numpy == 1.11.3; python_version == "2.6" or (python_version >= "3.2" and python_version <= "3.3")',
+    'numpy >= 2.0.0; python_version >= "3.9"',
```

Comment:
I suggest here `    'numpy == 2.0.0',`:
- We need `==` and not `>=` because this is `numpy` as build requirement, and we want to pin to the oldest supported `numpy` version.
- Since the PR effectively sets the minimum supported Python version to 3.9, we do not need anymore the block `; python_version >= "3.9"`.
- Consequently with the previous point, I would also suggest that you edit the `setup.py` file and set `"python_requires": ">=3.9, <3.13",`.